tests/Bridge/Doctrine/Persister/ObjectManagerPersisterTest.php

```diff
@@ -140,6 +140,24 @@ public function testCanPersistMultipleEntitiesWithExplicitIdentifierSet(): void
         self::assertInstanceOf(DummyWithIdentifier::class, $entity);
     }
 
+    public function testClearingUnitOfWorkPersister(): void
+    {
+        $dummy = new DummyWithProperty();
+        $dummy->id = 1;
+
+        $this->persister->persist($dummy);
+        $this->persister->flush();
+
+        $dummy = new DummyWithProperty();
```

Comment:
I think you're missing the import statement

Reply:
done